src/pages/summer/Login/LoginInput/LoginInput.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+
+import './LoginInput.scss';
+
+class LoginInput extends React.Component {
+  state = {
+    idInput: '',
+    pwInput: '',
+    idVld: false,
+    pwVld: false,
+  };
+
+  gotoMain = e => {
+    const { IP_ADDRESS } = this.props;
+    e.preventDefault();
+
+    fetch(`http://${IP_ADDRESS}:8000/users/signin`, {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.idInput,
+        password: this.state.pwInput,
+        // name: '포켓몬 지우',
+        // phone_number: '010-0000-0000',
+      }),
+    })
+      .then(res => res.json())
+      .then(res => {
+        localStorage.setItem('TOKEN', res.access_token);
+        this.props.history.push('/summer/main');
+      });
+  };
+
+  handleInput = e => {
+    const { value, name } = e.target;
+
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  checkLoginVld = () => {
+    const { idInput, pwInput } = this.state;
+
+    const checkIdVld = /^(?=.*[@])[a-z0-9_.@]{5,}$/gi;
+    const checkedId = checkIdVld.test(idInput);
+    const checkPwVld = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{5,}$/;
+    const checkedPw = checkPwVld.test(pwInput);
+
+    checkedId
+      ? this.setState({ idVld: true })
+      : this.setState({ idVld: false });
+    checkedPw
+      ? this.setState({ pwVld: true })
+      : this.setState({ pwVld: false });
```

Comment:
```suggestion
    this.setState({ isIdValid, isPwValid })
```
위에 코멘트 드린 내용들을 쭉 적용해보면 최종적으로 이렇게 하나로 줄일 수 있을 것 같습니다

Reply:
> 위에 코멘트 드린 내용들을 쭉 적용해보면 최종적으로 이렇게 하나로 줄일 수 있을 것 같습니다

이 부분을 정말 줄이고 싶었는데 감사합니다!